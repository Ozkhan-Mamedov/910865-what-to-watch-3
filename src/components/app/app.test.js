import React from "react";
import renderer from "react-test-renderer";

import App from "./app";

const promoFilmData = {
  name: `name`,
  genre: `genre`,
  releaseDate: `date`,
};

const films = [
  {
    name: `Film#1`,
    picture: `picture#1`,
    genre: `genre#1`,
    id: `#1`,
  },
  {
    name: `Film#2`,
    picture: `picture#2`,
    genre: `genre#2`,
    id: `#2`,
  },
  {
    name: `Film#3`,
    picture: `picture#3`,
    genre: `genre#3`,
    id: `#3`,
  },
];

it(`App component renders correctly`, () => {
  const tree = renderer
    .create(<App promoFilmData={promoFilmData} films={films} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
