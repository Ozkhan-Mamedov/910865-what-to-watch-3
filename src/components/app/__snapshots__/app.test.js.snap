// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App with authorization renders correctly 1`] = `null`;

exports[`App without authorization component renders correctly 1`] = `null`;
